src/cascadia/Remoting/Monarch.cpp

```diff
@@ -20,6 +21,11 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
     Monarch::Monarch() :
         _ourPID{ GetCurrentProcessId() }
     {
+        try
+        {
+            _desktopManager = winrt::create_instance<IVirtualDesktopManager>(__uuidof(VirtualDesktopManager));

```

Comment:
Do we need a new one each time a monarch is created? Or should we make `_desktopManager` static, and only create it if we don't have one?

Reply:
Yea, we do, because there's only ever one Monarch pre-process. So the next process can't really re-use the one from the previous process 😉